client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,155 @@
+import { useHistory } from 'react-router-dom';
+
+import useStyles from './useStyles';
+
+const ManageBookings = (): JSX.Element => {
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+
+  const history = useHistory();
+  const classes = useStyles();
+
+  useEffect(() => {
+    initSocket();
+  }, [initSocket]);
+
+  const [date, changeDate] = useState<MaterialUiPickersDate>(new Date());
+  const [isSelectedDate, setIsSelectedDate] = useState<boolean>(false);
+
+  const [requests, setRequests] = useState<Array<BookingRequest>>(mockBookingRequest);
+  const nextBooking = requests.reduce((request, closest) =>
+    new Date(request.start.getTime() - new Date().getTime()) < closest.start &&
+    request.start &&
+    request.start > new Date()
+      ? (closest = request)
+      : (closest = closest),
+  );
+  const [selectedBooking, setSelectedBooking] = useState<BookingRequest | undefined>(nextBooking);
+
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
+    history.push('/login');
+    // loading for a split seconds until history.push works
+    return <CircularProgress />;
+  }
+
+  const handleDateChange = (e: MaterialUiPickersDate) => {
+    if (!e) return;
+    changeDate(e);
+    setIsSelectedDate(true);
+    const request = requests.find(
+      (request) => Intl.DateTimeFormat('en').format(request.start) === Intl.DateTimeFormat('en').format(e),
+    );
+    setSelectedBooking(request);
+  };
+
+  const renderDay = (
+    day: MaterialUiPickersDate,
+    selectedDate: MaterialUiPickersDate,
+    dayInCurrentMonth: boolean,
+    dayCompmonent: JSX.Element,
+  ) => {
+    const component =
+      day &&
+      Intl.DateTimeFormat('en').format(nextBooking.start) ===
+        Intl.DateTimeFormat('en').format(new Date(day.toString())) ? (
+        <Avatar className={classes.upcomingBookingCircle}>{dayCompmonent}</Avatar>
+      ) : (
+        <span>{dayCompmonent}</span>
+      );
+    return component;
+  };
+
+  return (
+    <Grid container className={classes.root}>
+      <Grid container xs={6} className={classes.bookingsPart}>
+        <Paper key="selectedBookingPaper" className={classes.bookingPaper}>
+          <Typography className={classes.selectedBooking}>
+            {isSelectedDate ? 'BOOKINGS ON SELECTED DATE:' : 'YOUR NEXT BOOKING:'}
+          </Typography>
+          {(selectedBooking && (
+            <Booking
+              status={selectedBooking.status}
+              start={new Date(selectedBooking.start)}
+              end={new Date(selectedBooking.end)}
+              sitterId={selectedBooking.sitterId}
+            />
+          )) || (
+            <Paper>
+              <Grid className={classes.noBooking}>
+                <Typography>No bookings on selected date</Typography>
+              </Grid>
+            </Paper>
+          )}
+        </Paper>
+        <Paper key="currentBookingsPaper" className={classes.bookingPaper}>
```

Comment:
The `key` prop isn't needed here, it only helps React render a list of components (more info on `key` [here](https://reactjs.org/docs/lists-and-keys.html#keys). Please remove any unnecessary `key` props used throughout the code.

Also keep in mind that we use hyphenated key names just like html ids and route names 👍🏼